### ISSUE TITLE
Add demo image or video

### ISSUE THREAD

**Issue by hinell** [inkarkat/vim-EntryComplete#2]
**Motivation**

To better understand how it looks like in the wild.

**Comment by inkarkat** [comment]
Yes, that would be nice. But with more than 100 published plugins, I won't have the time for that myself, unfortunately, but I'm open to contributions.
